_posts/2019/2019-09-17-rnd-day-2019-eating-pizza-in-the-cloud.md

```diff
@@ -0,0 +1,73 @@
+authors: ["Birgit"]
+about_authors: ["bbader"]
+---
+
+Every year during this time we organize our R&D Day.
+Developers from all locations gather togehter and spend a day on topics that are currently relevant to them or might be exciting to work on.
+During the last two months, all colleagues had the opportunity to suggest topics they would like to learn more about or bring in something they want share themselves.
+With the help of our Scrum Masters this converted into a one-day agenda with a nice mixture of talks, workshops, and open spaces.
+Spoiler alert: and of course there was no way around pizza 🍕!
+
+## Biased remote work in the cloud
+
+Our Scrum Master started off with a great talk about "Unconcious Bias".
+This was so eye-opening!
+After a small intro we were split into groups to brainstorm solutions together, and discuss recommendations on what to do with a specific unconcious bias situation.
+There were examples such as *A coworker asks: "Who's the new girl?"* or *You're in a meeting and a female colleague is spoken over or interrupted.*
+
+One of our JavaScript developers continued with a talk about remote work.
+This was specifically interesting to us, because we have development teams in different locations and several teams have remote colleagues.
+So what's the good and the bad about remote work?
+What's the challenge when working from home?
+Is it only the TV and the fridge that can distract us from focusing on a topic?
+Can we overcommunicate?
+There were many suggestions we can reflect on and things we can improve.
+
+"The last year in the cloud" was a great recap held by one of our devops engineers.
+What went well?
+Anything new?
+What did we say goodbye to?
+What expenses did we have for cloud services?
+Interesting talk that also non-devs were able to follow.
+
+## Clean and social: surviving in a multi-tenancy shell on GitHub
+
+The afternoon was blocked for short talks, workshops, and open spaces.
+Many sessions had already been prepared upfront, such as *Surviving in a shell*, *Social Engineering*, *Clean Architecture*, *GitHub for beginners*, or *Beyond multi-tenancy*.
+
+{% twitter https://twitter.com/donaldlika1/status/1171782960300875776 %}
+
+Some colleagues came up with spontaneous open space sessions.
+You could for example attend *GitHub for grown ups*, *Insights into the development of the esitekic app*, *GraphQL*, or *Development Feedback Loops*.
+
+All the topics were super interesting, and there was a lot to learn.
+
+## Any thoughts? Many!
+
+From the *GitHub for beginners* workshop that I held together with my colleague, we gathered some statements about the R&D Day.
+After a 90 minutes workshop, our newborn GitHubbies (all non-devs!) were able to make a Pull Request to our [epages-devportal](https://github.com/ePages-de/epages-devportal){:target="_blank"}.
+We asked them to commit some thoughts about the day, and here we go with some quotes (they have not been checked for grammar 😆):
```

Comment:
Not sure if the "they have not been checked for grammar" part might offend some authors as they might think that their English is bad... But it's more the fact that they did it in a rush and didn't even know that it will be published and not the fact that they don't know better?

(Plus: should we ask them if we can publish their quotes?)